src/firmware_new/boards/frankie_v1/io/drivetrain.c

```diff
@@ -0,0 +1,48 @@
+#include "firmware_new/boards/frankie_v1/io/drivetrain.h"
+
+#include <assert.h>
+#include <stdbool.h>
+
+static DriveTrainUnit_t *_front_left_drive_unit;
+static DriveTrainUnit_t *_front_right_drive_unit;
+static DriveTrainUnit_t *_back_left_drive_unit;
+static DriveTrainUnit_t *_back_right_drive_unit;
+
+static bool initialized = false;
+
+void io_drivetrain_init(DriveTrainUnit_t *front_left_drive_unit,
+                        DriveTrainUnit_t *front_right_drive_unit,
+                        DriveTrainUnit_t *back_left_drive_unit,
+                        DriveTrainUnit_t *back_right_drive_unit)
```

Comment:
Nit again:

Usually we keep the '*' on the type

Reply:
See above comment reply